File: PYTHON/Human.py
class Human:
    # kelas untuk menampung data human dan method mangan

    # atribut
    __nik = ""
    __nama = ""
    __jenisKelamin = ""

    # konstruktor
    def __init__(self):
        self.__nik = ""
        self.__nama = ""
        self.__jenisKelamin = ""

    # setter getter
    def setNik(self, nik):
        self.__nik = nik
    def setNama(self, nama):
        self.__nama = nama
    def setJenisKelamin(self, kelamin):
        self.__jenisKelamin = kelamin
    
    def getNik(self):
        return self.__nik
    def getNama(self):
        return self.__nama
    def getJenisKelamin(self):
        return self.__jenisKelamin
    
    # method
    def makan(self):
        print("oke mangan..")

File: CPP/Human.cpp
#include <iostream>
#include <string>

using namespace std;

class Human {
    // attribute
    private:
        string nik, nama, jenisKelamin;
    
    // constructor
    public:
    Human() {
        this->nik = "";
        this->nama = "";
        this->jenisKelamin = "";
    }

    Human(string nik, string nama, string jenisKelamin) {
        this->nik = nik;
        this->nama = nama;
        this->jenisKelamin = jenisKelamin;
    }

    // getter and setter
    string getNik() {
        return this->nik;
    }

    string getNama() {
        return this->nama;
    }

    string getJenisKelamin() {
        return this->jenisKelamin;
    }

    void setNik(string nik) {
        this->nik = nik;
    }

    void setNama(string nama) {
        this->nama = nama;
    }

    void setJenisKelamin(string jenisKelamin) {
        this->jenisKelamin = jenisKelamin;
    }

    // method
    void makan() {
        cout << "oke mangan.." << endl;
    }

    void printHuman() {
        cout << "NIK : " << this->nik << endl;
        cout << "Nama : " << this->nama << endl;
        cout << "Jenis Kelamin : " << this->jenisKelamin << endl;
    }

    // destructor
    ~Human() {}
};
